### PR TITLE
Added utility functions to update and remove a view

### PR DESCRIPTION
**Pull request by tkakar**
This PR adds functions `get_views()`, `get_view_by()`, and `remove_view_by` to update or remove a view from the ConfigView.
Addresses #104

### CODE REVIEW

**Review by NickAkhmetov — COMMENTED** [review]
Could we add some tests for these functions? e.g. extending the cases here: https://github.com/vitessce/vitessce-python/blob/tkakar/cat-907-add-get-view/tests/test_config.py

**Comment by tkakar** [comment]
@NickAkhmetov thanks for reviewing. Will add the tests.

File: vitessce/config.py [hunk 1]
```diff
@@ -1495,6 +1495,65 @@ def to_dict(self, base_url=None):
...
+    def get_views(self):
+        """
+        Provides all the views in the config.layout object list
+
+        :returns: A list of VitessceConfigView objects.
+
+        """
+        return self.config["layout"]
+
+    def get_view_by(self, view_by):
```
Comment (keller-mark) [inline vitessce/config.py:1507]:
```suggestion
    def get_view_by_index(self, view_by):
```

I would split this into two functions, `get_view_by_index` and `get_first_view_by_type`

to make it more explicit the functionality

File: vitessce/config.py [hunk 2]
```diff
@@ -1495,6 +1495,65 @@ def to_dict(self, base_url=None):
...
+        elif isinstance(view_by, str):
+            for view in self.config["layout"]:
+                # TODO: returns the first match, do we need to return all matches in case of component type (scatterplots)
+                if view.view["component"].lower() == view_by.lower():
+                    return view
+            raise ValueError(f"No view found with component view_by: {view_by}")
+        else:
+            raise TypeError("view_by must be an integer index or a string representing the view type")
+
+    def remove_view_by(self, view_by):
```
Comment (keller-mark) [inline vitessce/config.py:1531]:
```suggestion
    def remove_view_by_index(self, view_by):
```

Similarly I would split this into two as well, `remove_view_by_index` and `remove_first_view_by_type`

**Comment by keller-mark** [comment]
It looks very good overall, I would change the naming of these functions and refactor the two into four for clarity, then it is ready to merge

**Comment by tkakar** [comment]
@keller-mark thanks for reviewing. will fix these.

**Review by keller-mark — APPROVED** [review]

**Pull request merged**